src/content/docs/licenses/open-source/code-of-conduct-reporting.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: New Relic Open Source Community Code of Conduct - Reporting Guide
```

Comment:
@ubanerjeeNR We use sentence case even in titles (https://docs.newrelic.com/docs/style-guide/capitalization/capitalization/#heading_caps)

I think this should be: "New Relic open source community code of conduct - reporting guide"

Reply:
fixed